src/main/kotlin/no/nav/familie/klage/behandling/domain/Behandling.kt

```diff
@@ -35,6 +36,7 @@ data class Behandling(
 
 data class PåklagetVedtakDetaljer(
     val fagsystemType: FagsystemType,
+    val vedtakType: VedtakType,
```

Comment:
Forslag:
1) Ikke ha to felt her 
2) Bytt `FagsystemType` med `VedtakType` som type :) 
3a) Vurdér om vi skal behholde navnet fagsystemType  
3b) Migrering som renamer fagsystemType til vedtakType

3b gir "nedetid" for oppslag av eksisterende behandling i ca. 10sek til 1 minutt.